.azure-pipelines/client.yml

```diff
@@ -0,0 +1,180 @@
+
+
+      - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+        displayName: 'Sign KeyVault Assemblies'
+        inputs:
+          ConnectedServiceName: 'ESRP Signing Service'
+
+          FolderPath: 'src\SDKs'
+
+          Pattern: '$(sign.pattern)'
+
+          signConfigType: inlineSignParams
+
+          inlineOperation: |
+           [
+               {
+                   "keyCode": "CP-233863-SN",
+                   "operationSetCode": "StrongNameSign",
+                   "parameters": [ ],
+                   "toolName": "sign",
+                   "toolVersion": "1.0"
+               },
+               {
+                   "keyCode": "CP-233863-SN",
+                   "operationSetCode": "StrongNameVerify",
+                   "parameters": [ ],
+                   "toolName": "sign",
+                   "toolVersion": "1.0"
+               },
+               {
+                   "keyCode": "CP-230012",
+                   "operationSetCode": "SigntoolSign",
+                   "parameters": [
+                   {
+                       "parameterName": "OpusName",
+                       "parameterValue": "Microsoft"
+                   },
+                   {
+                       "parameterName": "OpusInfo",
+                       "parameterValue": "http://www.microsoft.com"
+                   },
+                   {
+                       "parameterName": "FileDigest",
+                       "parameterValue": "/fd \"SHA256\""
+                   },
+                   {
+                       "parameterName": "PageHash",
+                       "parameterValue": "/NPH"
+                   },
+                   {
+                       "parameterName": "TimeStamp",
+                       "parameterValue": "/tr \"http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer\" /td sha256"
+                   }
+                   ],
+                   "toolName": "sign",
+                   "toolVersion": "1.0"
+               },
+               {
+                   "keyCode": "CP-230012",
+                   "operationSetCode": "SigntoolVerify",
+                   "parameters": [ ],
+                   "toolName": "sign",
+                   "toolVersion": "1.0"
+               }
+           ]
+
+        condition: and(succeeded(), notIn(variables['Build.Reason'], 'PullRequest'))
+
+      - task: MSBuild@1
+        displayName: 'Build NuGet Packages'
+        inputs:
+          solution: build.proj
+
+          #TODO: change the target to just "Package" once it's merged into the main bootstrap flow
+          msbuildArguments: '/t:CategorizeProjects,Package /p:Scope=$(source.scope) /p:Configuration=Release /flp:v=diag /flp:Summary;Verbosity=minimal;LogFile=PkgBuild.sum /flp1:warningsonly;logfile=PkgBuild.wrn /flp2:errorsonly;logfile=PkgBuild.err /p:DebugOutput=true /p:DebugTrace=$(system.debug)'
```

Comment:
@kurtzeborn after your https://github.com/Azure/azure-sdk-for-net/pull/5066, you do not need to append the CategorizeProjects target. Only executing Package should suffice.

Reply:
Yep.  Changing it to just "Package"

Reply:
@kurtzeborn with the changes done in #5076 you will have to start using PackageNoBuild instead of Package.
Here is the change
https://github.com/Azure/azure-sdk-for-net/pull/5076/files#diff-bb2b2238ee3ac486c97115af68c714e3R265

Reply:
Yep, I'll make the update.